testing/fake-dom.js

```diff
@@ -253,6 +253,11 @@ export class FakeWindow {
      * @const
      */
     this.requestAnimationFrame = raf;
+
+    // Styles.
+    this.getComputedStyle = function() {
+      return window.getComputedStyle.apply(window, arguments);
```

Comment:
Good find. Could it be that you want
`arguments[0].defaultView` instead of `window`?

Reply:
Maybe :) `arguments[0]` is the element here, so I tried `arguments[0].ownerDocument.defaultView` which points at the fake win, but it throws an Illegal invocation error.

Reply:
This would be expected to work when the element is from the actual window.